tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
+    "target": "ES2019",
```

Comment:
So as it turns out Node.JS 13.9 only has nullish coalescing and optional chaining working with an optional harmony V8 flags.